apps/shared/src/themes-mui/buildMuiTheme.ts

```
@@ -0,0 +1,101 @@
+// =============================================================================
+// MUI Theme Bridge
+// =============================================================================
+
+import { createTheme, type Theme } from '@mui/material/styles';
+import type { ThemeTokens } from 'shell';
```

ESLint

[error] 'shell' import is restricted from being used. Renamed: import from 'shell'.

(no-restricted-imports)